### DISCUSSION TITLE
[Bug] Incorrect status emoji on mobile

### DISCUSSION THREAD

**Discussion by Deadlyelder** [community/community#8189] [Mobile]
I have the following status set on my Github user:

<img width="353" alt="Screen Shot 2021-11-28 at 10 36 00 AM" src="https://user-images.githubusercontent.com/6140893/143762809-9f7c2cd1-8aa6-4bcb-8aef-52a2ce160b24.png">

When visiting from the mobile, it tends to be messed up:

<img width="371" alt="Screen Shot 2021-11-28 at 10 37 23 AM" src="https://user-images.githubusercontent.com/6140893/143762845-c61f3594-b2ee-4ea8-b9b4-dc3fcde57245.png">

I see the similar issue with other Github user profile as well, especially when setting up the Github specific emojis.

**Comment by 2910418383099207555** [comment]
Compass calibration not found

**Comment by eliperkins** [comment]
Thanks for this bug report @Deadlyelder! That's definitely not how it's supposed to look on mobile 😅 

I've filed a bug report in our internal tracking issues to get this resolved. Stay tuned!
